main.nf

```diff
@@ -1,34 +1,86 @@
+/*
+ * This Source Code Form is subject to the terms of the GNU GENERAL PUBLIC LICENCE
+ * License, v. 3.0. 
+ */
+
+
+/* 
+ * 'PhIP-Flow' - A Nextflow pipeline for running common phip-seq analysis workflows
+ * 
+ * Fred Hutchinson Cancer Research Center, Seattle WA.
+ * 
+ * Jared Galloway
+ * Kevin Sung
+ * Sam Minot
+ * Erick Matsen 
+ */
+
+/* 
+ * Enable DSL 2 syntax
+ */
+nextflow.enable.dsl = 2
+
+/*
+ * Define the default parameters - example data get's run by default
+ */ 
+params.sample_table     = "$baseDir/data/pan-cov-example/sample_table.csv"
+params.peptide_table    = "$baseDir/data/pan-cov-example/peptide_table.csv"
+params.results          = "$baseDir/results/"
```

Comment:
I would not recommend writing outputs to the `$baseDir`, which is the cloned copy of this GitHub repository in the user's account

Reply:
Ah, nice catch - so If I always want to write relative to working directory at execution, should they all be relative to $PWD?

Or are all paths relative to execution regardless? I've always been a little unclear on this. Both seem to work when I test.

Reply:
If you leave out any prefix, the path will be relative to the working directory. If you were executing the workflow from the main directory of the repository, then the behavior may have been identical during testing.

For this workflow, I think it would make sense to set the default `sample_table` and `peptide_table` to the example data (since they are read-only), while the `results` should probably be pointed towards the working directory. As a general rule, I don't like to set up workflows which write to the directory which contains the repository.